packages/desktop/views/onboarding/views/create-profile/views/ChooseCreateProfileFlowView.svelte

```diff
@@ -27,8 +27,10 @@
 
     async function onProfileTypeClick(createProfileType: CreateProfileType): Promise<void> {
         isBusy = { ...isBusy, [createProfileType]: true }
-        const type = createProfileType === CreateProfileType.Ledger ? ProfileType.Ledger : ProfileType.Software
-        updateOnboardingProfile({ createProfileType, type })
+        const isLedgerProfile = createProfileType === CreateProfileType.Ledger
+        const type = isLedgerProfile ? ProfileType.Ledger : ProfileType.Software
+        const strongholdVersion = isLedgerProfile ? undefined : $onboardingProfile.strongholdVersion
```

Comment:
I would improve this logic here, becasue you are either taking the value that already exists, or undefining it, and use it to update the value that already exists. There shouldnt be a stronghold version in the onboarding profile at a this point, so you should set it here, eg
```ts
        const isLedgerProfile = createProfileType === CreateProfileType.Ledger
        const type = isLedgerProfile ? ProfileType.Ledger : ProfileType.Software
        const strongholdVersion = isLedgerProfile ? undefined : STRONGHOLD_VERSION
        updateOnboardingProfile({ createProfileType, type, ...(strongholdVersion && { strongholdVersion }) })
```